holoviews/plotting/bokeh/__init__.py

```diff
@@ -156,6 +156,7 @@ def colormap_generator(palette):
     options.Points = Options('plot', show_frame=True)
 
 options.Histogram = Options('style', line_color='black', fill_color=Cycle())
+options.Distribution = Options('style', muted_alpha=0.2)
```

Comment:
Sorry, should have been clearer, I meant adding this to the ``Histogram`` options one line above.

Reply:
give me a sec